orttraining/orttraining/test/python/orttraining_test_python_bindings.py

```diff
@@ -226,10 +230,19 @@ def test_training_module_checkpoint():
         checkpoint_save_path = os.path.join(temp_dir, "checkpoint_export.ckpt")
 
         model.save_checkpoint(checkpoint_save_path)
+        old_params = model.get_contiguous_parameters()
 
-        # TODO : Load checkpoint to a zeroed model and assert parameters are different.
+        # Assert the checkpoint was saved.
```

Comment:
looks this change did complete the TODO "TODO : Load checkpoint to a zeroed model and assert parameters are different."

Reply:
the change I did will test the same thing, there's no need to create a zeroed model and load it to test if the checkpoint params remain the same.